City-of-Helsinki/helsinki-paatokset
# PP-285: Add policymaker ref to trustees + display it on page

Add a new field for policymaker reference to trustees. Add logic to display it on the page.

To test:

- Checkout to branch. Run `make drush-cr; make drush-cim`
- Edit a trustee. Add a reference to a policymaker through the _policymaker_reference_ field.
- View the trustee page. Information about the reference should be visible at the bottom of the trustee info box, like this: `[trustee_name] is [policymaker]`
